Library/Homebrew/cmd/style.rb

```diff
@@ -111,12 +111,21 @@ def check_style_impl(files, output_type, options = {})
       args << "--only" << cops_to_include.join(",")
     end
 
+    has_non_formula = files.nil? || files.any? do |file|
+      puts "FILE: #{file}"
+      File.expand_path(file).start_with? HOMEBREW_LIBRARY_PATH
+    end
+    config_file = if has_non_formula
+      HOMEBREW_LIBRARY_PATH/(ARGV.include?("--rspec") ? ".rubocop-rspec.yml" : ".rubocop.yml")
```

Comment:
@apjanke I don't think the ternary operator is worth having just to avoid duplicating `HOMEBREW_LIBRARY_PATH/`. `if has_formula ... elif ARGV.include?("--rspec") ... else` would be clearest, I think (but involve reversing the `has_non_formula` to `has_formula` which I think would also be clearer).

Reply:
`has_non_formula` doesn't reverse to `has_formula`, it reverses to `doesnt_have_non_formula` or `all_are_formula`, and the `if/then` logic is not equivalent to `if has_formula ... elif ARGV.include?("--rspec") ... else`. (Because there might be a mix of formula and non-formula files, and the way it's coded, the presence of non-formula files is the determining factor.)

I removed the ternary operator, though.